contents/privacy.md

```diff
@@ -209,18 +226,26 @@ We will hold all the data for 24 months. Prior to that, your personal informatio
 
 ### Your rights and choices as an applicant
 
-You may access, correct, or delete your personal information at any time. You may also withdraw your consent, or object to the processing of your personal information when based on our legitimate interests. To do so, please email us at: [careers@posthog.com](mailto:careers@posthog.com)
+Please see the section on Global Privacy Practices and Your Rights above.
 
 ## Contacting PostHog about your privacy
 
-If you have questions or concerns about the way we are handling your information, or would like to exercise your privacy rights, please email us with the subject line "Privacy Concern" at [privacy@posthog.com](mailto:privacy@posthog.com).
+The relevant data controller for any personal information processed in connection with our Websites or self-managed installations is PostHog Inc, 2261 Market Street #4008, San Francisco, CA 94114. If you apply for a job with us, the relevant data controller is the PostHog entity which will be your employer. 
```

Comment:
```suggestion
The relevant data controller for any personal information processed in connection with our Websites or self-managed installations is PostHog Inc, 2261 Market Street #4008, San Francisco, CA 94114. If you apply for a job with us, the relevant data controller is the country-specific PostHog entity which will be your employer. 
```

Reply:
felt above was a little vague (ie this could infer PostHog Inc, which isn't your employer if you're int he UK)